example/storybook/src/ui-components/Input/styled-components/Icon.tsx

```diff
@@ -5,9 +5,9 @@ export default styled(
   Pressable,
   {
     justifyContent: 'center',
-    alignItems: 'center',
+    alignSelf: 'center',
+    my: 'auto',
```

Comment:
`align-self` and `my: auto` is not good practice. 
use justifyConent:center and alignItems:'center'